docs/ossm/injection/README.md

```diff
@@ -49,81 +49,183 @@ The injector is configured with the following logic:
+
+This example also demonstrates the use of a revision label for sidecar injection. In this case, the `Istio` resource has been created with the name "my-mesh". A unique resource `Istio` name is needed when there are multiple Istio control planes present in the same cluster, or a revision based control plane upgrade is in progress.
+
+Procedure:
+
+1. Verify the revision name of the Istio control plane:
+
+    ```console
+    $ oc get istiorevision
+    NAME      TYPE    READY   STATUS    IN USE   VERSION   AGE
+    my-mesh   Local   True    Healthy   False    v1.23.0   47s
+    ```
+    Since the revision name is `my-mesh`, we must use the a revision label to enable sidecar injection. In this case, `istio.io/rev=my-mesh`.
+
+1. For workloads already running, verify that they show "1/1" containers as "READY", indicating that the pods are currently running without sidecars:
+
     ```bash
-    oc delete pod -l app=sleep
+    $ oc get pods -n bookinfo
+    NAME                             READY   STATUS    RESTARTS   AGE
+    details-v1-65cfcf56f9-gm6v7      1/1     Running   0          4m55s
+    productpage-v1-d5789fdfb-8x6bk   1/1     Running   0          4m53s
+    ratings-v1-7c9bd4b87f-6v7hg      1/1     Running   0          4m55s
+    reviews-v1-6584ddcf65-6wqtw      1/1     Running   0          4m54s
+    reviews-v2-6f85cb9b7c-w9l8s      1/1     Running   0          4m54s
+    reviews-v3-6f5b775685-mg5n6      1/1     Running   0          4m54s
     ```
-1. Verify a new pod is created with the injected sidecar. The original pod has `1/1 READY` containers, and the pod with injected sidecar has `2/2 READY` containers.
+
+1. To find your deployments use the oc get command. For example, to view the Deployment YAML file for the 'ratings-v1' microservice in the bookinfo namespace, use the following command to see the resource in YAML format.
+
     ```bash
-    oc get pod -l app=sleep
-    NAME                     READY   STATUS    RESTARTS   AGE
-    sleep-5577c64d7c-w9vpk   2/2     Running   0          12s
+    oc get deployment -n bookinfo ratings-v1 -o yaml
+    ```
+
+1. Open the application’s Deployment YAML file in an editor.
+
+1. Update the `spec.template.metadata.labels` section of your Deployment YAML file to include the appropriate pod injection or revision label. In this case, `istio.io/rev: my-mesh`:
+
+    ```yaml
+    kind: Deployment
+    apiVersion: apps/v1
+    metadata:
+    name: ratings-v1
+    namespace: bookinfo
+    labels:
+      app: ratings
+      version: v1
+    spec:
+      template:
+        metadata:
+          labels:
+            istio.io/rev: my-mesh
     ```
-1. View the detailed state of the injected pod. You should see the injected `istio-proxy` container.
+1. If the Deployment was already running, it will need to be restarted for sidecar injection to occur:
```

Comment:
The same as above, the workload is automatically rollout after editing the deployment so this step is not needed.

Reply:
Correct, will update.